docs/config.rst

```diff
@@ -11,20 +11,69 @@ provided as configuration options, and finally falls back to the default values
 Java Kafka clients.
 
 
+``bootstrap.servers``
+  A list of host/port pairs to use for establishing the initial connection to the Kafka cluster. The client will make use of all servers irrespective of which servers are specified here for bootstrapping—this list only impacts the initial hosts used to discover the full set of servers. This list should be in the form host1:port1,host2:port2,.... Since these servers are just used for the initial connection to discover the full cluster membership (which may change dynamically), this list need not contain the full set of servers (you may want more than one, though, in case a server is down).
+
+  * Type: string
+  * Default: "PLAINTEXT://localhost:9092"
+  * Importance: high
+
 ``id``
   Unique ID for this REST server instance. This is used in generating unique IDs for consumers that do not specify their ID. The ID is empty by default, which makes a single server setup easier to get up and running, but is not safe for multi-server deployments where automatic consumer IDs are used.
 
   * Type: string
   * Default: ""
   * Importance: high
 
+``listeners``
+  List of listeners. http and https are supported. Each listener must include the protocol, hostname, and port. For example: http://myhost:8080,https://0.0.0.0:8081
+
+  * Type: list
+  * Default: []
+  * Importance: high
+
 ``schema.registry.url``
   The base URL for the schema registry that should be used by the Avro serializer.
 
   * Type: string
   * Default: "http://localhost:8081"
   * Importance: high
 
+``ssl.key.password``
```

Comment:
I guess you were intending to use these to support TLS for the proxy itself? If so, maybe we could move them into a separate patch?